lib/cadet/accounts/notification_type.ex

```diff
@@ -10,5 +10,7 @@ defenum(Cadet.Accounts.NotificationType, :notification_type, [
   # Notifications for unsubmitted submissions
   :unsubmitted,
   # Notifications for submitted assessments
-  :submitted
+  :submitted,
+  # Notifications for unpublished grades
+  :unpublishedGrading
```

Comment:
let's keep the atom as snake case, as camel case is used for module names which are under the hood atoms.